### ISSUE TITLE
catkin_lint errors

### ISSUE THREAD

**Issue by gavanderhoorn** [VictorLamoine/ensenso_n10_description#2]
```
ensenso_eef: error: unconfigured build_depend on 'roslaunch'
```

**Comment by VictorLamoine** [comment]
Fixed with 8479b6522aefbf803b719a7264d66b25690eeea4, not sure if that's the proper way to do it!

**Comment by gavanderhoorn** [comment]
You no longer have any roslaunch tests, so you don't need the `build_depend` on `roslaunch`.
